Python/UpeachyaKansakar/tasks/task2/storeToPostgres.py

```diff
@@ -0,0 +1,31 @@
+import csv
+import psycopg2
+
+def main():
+    conn = psycopg2.connect("host=localhost dbname=python_training user=postgresuser password=root")
+    cur = conn.cursor()
+
+    # NOTE: Query for creating table
+    cur.execute("""
+        CREATE TABLE articles(
+        title text,
+        author text
+    )
+    """)
```

Comment:
Try to use context manager 
with connection.cursor() as cursor:
            cursor.execute(sql query)